x-pack/plugins/canvas/public/routes/workpad/hooks/use_fullscreen_presentation_helper.ts

```diff
@@ -4,16 +4,20 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { useContext, useEffect } from 'react';
+import { useCallback, useContext, useEffect } from 'react';
 import useEffectOnce from 'react-use/lib/useEffectOnce';
-import { usePlatformService } from '../../../services';
 import { WorkpadRoutingContext } from '..';
+import { coreServices } from '../../../services/kibana_services';
 
 const fullscreenClass = 'canvas-isFullscreen';
 
 export const useFullscreenPresentationHelper = () => {
   const { isFullscreen } = useContext(WorkpadRoutingContext);
-  const { setFullscreen } = usePlatformService();
+
+  const setFullscreen = useCallback(
```

Comment:
Is this callback even needed? How about just replacing `setFullscreen` with `coreServices.chrome.setIsVisible`

Reply:
Good call - done in 78ee22bf218055686f3659cf75fd8bfb7b494504